{"name": "Negative scenario's of API", "status": "passed", "steps": [{"name": "Given Getting the wrong url of dummy api", "status": "passed", "start": 1732167579888, "stop": 1732167579888}, {"name": "When sending the call to the server", "status": "passed", "start": 1732167579888, "stop": 1732167581154}, {"name": "Then validating the response status as 404", "status": "passed", "start": 1732167581154, "stop": 1732167581154}], "start": 1732167579888, "stop": 1732167581154, "uuid": "0e4f8133-4faa-4d78-b58d-753866c3e1c6", "historyId": "2af04a7e628a68e09e0f578e758cebd1", "fullName": "Validation of the API: Negative scenario's of API", "labels": [{"name": "severity", "value": "normal"}, {"name": "feature", "value": "Validation of the API"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}